sample-apps/deepgrow/main.py

```diff
@@ -97,12 +97,14 @@ def init_trainers(self) -> Dict[str, TrainTask]:
                 dimension=2,
                 roi_size=(256, 256),
                 model_size=(256, 256),
-                max_train_interactions=15,
+                max_train_interactions=10,
```

Comment:
Good luck to us on this one,

Reply:
We r not looking for validation dice.  And 2D model is better only if it asks me lesser than 10 clicks..

Reply:
I'm training the DeepEdit on 10 simulated clicks and it seems to work well